src/garage/torch/modules/mlp_module.py

```diff
@@ -0,0 +1,92 @@
+"""
+MLP Module.
+
+A Pytorch module composed only of a multi-layer perceptron (MLP), which maps
+real-valued inputs to real-valued outputs.
+"""
+
+from torch import nn as nn
+from torch.nn import functional as func
```

Comment:
Let's follow the pytorch documentations and do `from torch.nn import functional as F`